rich/cells.py

```diff
@@ -115,7 +115,7 @@ def chop_cells(text: str, max_size: int, position: int = 0) -> List[str]:
     characters = [
         (character, _get_character_cell_size(character)) for character in text
     ]
-    total_size = position + 1
+    total_size = position
```

Comment:
Always feels good to remove such occurrences of `+ 1` - they always feel a bit too magical, right? :smile: 
(and often not the right kind of magic :crystal_ball: :mage: :japanese_ogre: )